cmd/autothrottle/internal/api/api_test.go

```diff
@@ -18,16 +29,20 @@ func TestSetThrottle(t *testing.T) {
 	}
 
 	responseRecorder := httptest.NewRecorder()
-	handler := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) { throttleGetSet(w, req, zk) })
+	handler := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) { throttleGetSet(w, req, zk, trigger) })
 
 	// WHEN
 	handler.ServeHTTP(responseRecorder, req)
 
 	// THEN
 	checkResults(http.StatusOK, "throttle successfully set to 5MB/s, autoremove==false\n", responseRecorder, t)
+	if len(trigger) != 1 {
+		t.Errorf("mutation did not trigger config application, trigger channel lenght: %d", len(trigger))
```

Comment:
Typo (`lenght`) in this and a few of the other error messages in this test.

Also a similar nit to above asserting on the length of the channel, "better practice" to just consume the channel and record how many things we read.